docs/release-notes.md

```diff
@@ -2,6 +2,22 @@
 
 This page summarizes changes for each major update in Proton and Timeplus Cloud, including new features and important bug fixes.
 
+## Oct 30, 2023
+
+*Proton:*
+  * You can now install single native binary for Mac - check out our installation guide [here](https://github.com/timeplus-io/proton/wiki/Install-single-binary-Proton).
```

Comment:
.. for Mac or Linux

Reply:
Thanks @jovezhong, both changes resolved